### PR TITLE
Workaround: set long JWT token expiration

### PR DESCRIPTION
**Pull request by michaljurecko**
Changes:
- Temporary increase JWT token TTL.

### CODE REVIEW

File: internal/pkg/service/common/etcdclient/etcdclient.go [hunk 1]
```diff
@@ -168,6 +168,7 @@ func New(ctx context.Context, proc *servicectx.Process, tracer trace.Tracer, end
 		Username:             conf.username, // optional
 		Password:             conf.password, // optional
 		Logger:               etcdLogger,
+		PermitWithoutStream:  true, // always send keep-alive pings
```
Comment (michaljurecko) [inline internal/pkg/service/common/etcdclient/etcdclient.go:171]: Aby sa keep-alive requesty posielali stale, aj ked nie je este nastaveny ziaden watch.

File: provisioning/common/etcd/values.yaml [hunk 2]
```diff
@@ -62,7 +62,7 @@ auth:
     allowNoneAuthentication: false
   token:
     type: jwt
-    ttl: 10m
+    ttl: 10080m # temporary: https://github.com/etcd-io/etcd/pull/14995
```
Comment (michaljurecko) [inline provisioning/common/etcd/values.yaml:65]: V `etcd-client` - `3.5.6` sa pridavalo nejake watch retry, ked token expiroval, ... v `3.5.7` bol revert.
https://github.com/etcd-io/etcd/pull/14995 

My sme mali doteraz nastavene rotacie tokenu kazdych `10min`.
Docasne to nastavujem na `7dni`, kym sa mi to podari nastudovat, nasimulovat a vyriesit.

Comment (michaljurecko) [reply provisioning/common/etcd/values.yaml:65]: Vyzera to tak, ze worker sa po rotacii JWT tokenu nevie z toho dostat.

Comment (michaljurecko) [reply provisioning/common/etcd/values.yaml:65]: Vysledkom je zmena ENV:
image

**Review by JakubMatejka — APPROVED** [review]

**Pull request merged**